### ISSUE TITLE
Experiment time

### ISSUE THREAD

**Issue by ten-xi** [TesfayZ/CCM_MADRL_MEC#6]
Dear author, I am very interested in your work, may I ask how long you run an experiment?

**Comment by TesfayZ** [comment]
24 hours for the first, and 60 hours for the next two.

**Comment by ten-xi** [comment]
What if you put it on cuda? Have you tried that?

**Comment by ten-xi** [comment]
Run CCM_MADRL.py for 24 hours, so run the whole experiment

**Comment by ten-xi** [comment]
Can you provide comments for that part of the model?
  checkpoint = torch.load('./checkpoint/models_checkpoint'+str(self.InfdexofResult)+'.pth')    
        # Check for parameter differences in actors
        changes = []
        for agent_id in range(self.n_agents):
            ce = self.check_parameter_difference(self.actors[agent_id], checkpoint['actors'][agent_id])
            changes.append(ce)
        # Check for parameter differences in critics
        for agent_id in range(1):
            ce = self.check_parameter_difference(self.critics[agent_id], checkpoint['critics'][agent_id])
            changes.append(ce)
        if sum(changes) >1:
            #print("Model update detected", changes)
            self.save_models('./checkpoint/models_checkpoint'+str(self.InfdexofResult)+'.pth')
        elif sum(changes) == 1:
            print("No actor model update detected", changes)
            self.save_models('./checkpoint/models_checkpoint'+str(self.InfdexofResult)+'.pth')
            #exit()
        else:
            print("No model update detected", changes)
            self.save_models('./checkpoint/models_checkpoint'+str(self.InfdexofResult)+'.pth')
            #exit()
        '''     
         Is that the part?

**Comment by TesfayZ** [comment]
No, go to the last line and uncomment. The code you posted was used to
detect whether the weight of the neural network is updating with the
training. You don't need it  now.

**Comment by ten-xi** [comment]
Hello author, did you not consider putting these operations on the GPU at that time?If you run it on a GPU, is it much faster?
